python/pylibwholegraph/pyproject.toml

```diff
@@ -55,9 +55,13 @@ test = [
 [tool.rapids-build-backend]
 build-backend = "scikit_build_core.build"
 dependencies-file = "../../dependencies.yaml"
+matrix-entry = "cuda_suffixed=true"
 requires = [
     "cmake>=3.30.4",
     "cython>=3.0.0",
+    "libraft==25.6.*,>=0.0.0a0",
+    "librmm==25.6.*,>=0.0.0a0",
+    "libwholegraph==25.6.*,>=0.0.0a0",
```

Comment:
`pylibwholegraph` wheels should pick up a **runtime** dependency on `libwholegraph` wheels too, because you expect to be able to `import libwholegraph` in `pylibwholegraph/__init__.py`.

Reply:
Done